submissions.md

```diff
@@ -3,18 +3,63 @@ layout: default
 title: "ProbNum25 : Submissions"
 ---
 
-# Proceedings of ProbNum25
+
+# Call for Papers
+
+_Full details, including style files, will be published here in due course._
+
+## Proceedings of ProbNum25
 
 ProbNum25 will have a proceedings issue, published in the [Proceedings of Machine Learning Research (PMLR)](https://proceedings.mlr.press).
 Talks at ProbNum25 will be selected from among the submissions.
 Submissions can be up to 8 pages, with subjects broadly in the area of probabilistic computation.
```

Comment:
Can we say _probabilistic numerical computation_ here? (I'm not trying to be difficult at all; I just realised recently that many people take probabilistic computation to mean probabilistic programming, stochastic methods, etc.